components/eventing-controller/utils/jetstream_utils.go

```diff
@@ -24,7 +24,7 @@ func toJetStreamStorageType(s string) (nats.StorageType, error) {
 	return nats.MemoryStorage, fmt.Errorf("invalid stream storage type %q", s)
 }
 
-func toJetStreamRetentionPolicy(s string) (nats.RetentionPolicy, error) {
+func ToJetStreamRetentionPolicy(s string) (nats.RetentionPolicy, error) {
```

Comment:
(nit) Why do we need to move this file from `pkg/handlers` to `utils`? Can't we keep it in `pkg/handlers` and just export the methods? IMO, `utils` directory is for things more generic for the whole codebase, JetStream specific helpers should be in the `pkgs`.

Reply:
I moved them into `pkg/utils`. The reason I exposed them is because I want to reuse them in the EPP

Reply:
https://github.com/kyma-project/kyma/pull/13775#discussion_r838231841